README.md

```diff
@@ -1218,6 +1218,7 @@ The available fields are:
  - `repost_count` (numeric): Number of reposts of the video
  - `average_rating` (numeric): Average rating give by users, the scale used depends on the webpage
  - `comment_count` (numeric): Number of comments on the video (For some extractors, comments are only downloaded at the end, and so this field cannot be used)
+ - `download_count` (numeric): Number of times the video has been downloaded (by the platform's own downloader, not by YouTube Downloader Plus and/or any other YouTube Downloader deritives including itself)
```

Comment:
```suggestion
 - `download_count` (numeric): Number of times the video has been downloaded
```

1. > YouTube Downloader Plus
2. How can you be so sure? Who knows if TikTok counts 3rd party downloads, let alone other sites that might use this field